### ISSUE TITLE
Key.finger_all() does not consider local keys

### ISSUE THREAD

**Issue by moises-silva** [saltstack/salt#32377]
### Description of Issue/Question

The Key.finger_all() API is not taking into account the local keys.

PR coming for this, I just figured opening an issue would be better first. Let me know if going straight for the PR is preferred.

### Setup
Any fresh installation with minion keys already generated (e.g the minion started already).

### Steps to Reproduce Issue

In a minion you can run:

```python
from salt.config import minion_config
import salt.key

key = salt.key.Key(minion_config(None))
key.finger_all()
```
The minion key fingerprint won't be returned in the output.

### Versions Report
# salt --versions-report
Salt Version:
           Salt: 2016.3.0rc2-2-gb9d1138

Dependency Versions:
         Jinja2: 2.8
       M2Crypto: Not Installed
           Mako: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
         Python: 2.7.11 (default, Mar  3 2016, 11:00:04)
           RAET: Not Installed
        Tornado: 4.3
            ZMQ: 4.1.4
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
        libgit2: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.7
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed

System Versions:
           dist:
        machine: x86_64
        release: 4.1.20-1-lts
         system: Linux
        version: Not Installed

**Comment by jfindlay** [comment]
@moises-silva, generally an issue is not required but can be useful if discussion of a change needs to happen before creating the pull request.

**Comment by cachedout** [comment]
Closed via merge of #32379
